### DISCUSSION TITLE
Ubuntu Server 22.04.1 installer freezing after selecting "Install Ubuntu Server" (Mac M1 PRO Ventura 13.0, UTM 3.0.4)

### DISCUSSION THREAD

**Discussion by yusbarrett** [utmapp/UTM#4677] [UTM Help (macOS)] [answered]
**Describe the issue**  
So, I am trying to install Ubuntu Server 22.04.01, on my Mac M1 Pro, UTM 3.0.4, I've made sure to download the ARM version from the UTM Gallery, and after starting the virtual machine and selecting "Install Ubuntu Server" I get stuck in a frozen black screen with a cursor and nothing else happens (screenshot attached). I've tried the same with Fedora, and Ubuntu Desktop, with the same results. Checked the Console Crash Reports to search for any log, and it's empty.  
I then decided to download the x64 version and installed it using emulation, and had no issues installing, however, emulation is frustratingly slow. Is there any way you could recommend me to know why the ARM boot is freezing and not advancing, or how to fix this?  
I've seen similar threads with the same debug log warning (however it's a warning, not an error), and none of those threads seem to be consistent on a solution.
Thank you very much!

**Configuration**  
* UTM Version:  3.0.4
* macOS Version:  Ventura 13.0
* Mac Chip: M1 Pro 2021 

**Crash log**  
No crash log registered, log is empty.

**Debug log**  
qemu-aarch64-softmmu: warning: Spice: playback:0 (0x14c02b940): setsockopt failed, Operation not supported on socket
qemu-aarch64-softmmu: warning: Spice: record:0 (0x14c02b9f0): setsockopt failed, Operation not supported on socket

**Comment by yusbarrett** [comment]
<img width="797" alt="Screenshot 2022-11-14 at 20 30 38" src="https://user-images.githubusercontent.com/94024995/201842209-f456d507-ee3a-476f-b86c-0fbc02419d37.png">

**Reply by zayd17** [reply]
**<23> config noblink**

**Comment by gitKearney** [comment, marked as answer]
This was already solved here [#4173](https://github.com/utmapp/UTM/issues/4173#issuecomment-1179718559)

**Reply by yusbarrett** [reply]
Hi, gitKearney, thank you very much for your answer. 
I followed the instructions previously and what I got is the VM now displaying the UEFI Interactive Shell and the mapping table, since I'm still not entirely familiar with working on shell, where can I find the steps to go from here? Since I'm still unable to boot the installer and run the installation? 
Thanks!

<img width="796" alt="Screenshot 2022-11-15 at 16 46 57" src="https://user-images.githubusercontent.com/94024995/202042442-e44a3cef-a4cf-47e7-a5f1-e703e95b1b5d.png">

**Comment by eahicham** [comment]
Hey guys, I've been struggling with this problem for some time
macos 14; arm64 m1; utm 4.5.3 (and some version before).
The provided answer did not apply to my case since it supposed that the OS was installed and it was only matter of ejecting the cdrom before reboot.
I my case the ubuntu installer continuously crashes before the install is done (or sometimes even begins). I suspect  It has something to do we the network. I'm no sys expect but it looks to me that this problem has something to do with hardware (virtual) incompatibility.
the solution that worked from me was simply changing the system from QEMU 7.2 to QEMY 7.1. 
somehow this downgrade fixed the installation issue.

<img width="786" alt="image" src="https://github.com/user-attachments/assets/46eb31ff-cefc-4521-bf93-b79fe49b2f13">
